accepted/00102-git-integration.md

```diff
@@ -0,0 +1,63 @@
+- Feature Name: Git integration
+- Start Date: 2024-04-23
+
+# Summary
+[summary]: #summary
+
+This is a feature which will add a git integration to Uyuni without the need to use gitfs. It would be possible to sync formulas and playbooks and use them with an existent infrastructure.
+
+# Motivation
```

Comment:
I also think it would play nicely with EDA (https://github.com/uyuni-project/uyuni/discussions/7801)